packages/zig-wayland.json

```diff
@@ -0,0 +1,9 @@
+{
+  "author": "Isaac Freund",
+  "tags": [
+    "binding"
+  ],
+  "git": "https://github.com/ifreund/zig-wayland",
+  "root_file": "",
```

Comment:
Not sure what to put here as the root file is generated using a provided build step. See the README for details: https://github.com/ifreund/zig-wayland

Reply:
Hmm, good question… Maybe we should allow passing `null` here to allow to signal a more complicated setup. Same for @Snektron​s vulkan wrapper

Reply:
@ifreund Updated the repo to allow `null` to be passed into `root_file` (#19).

Reply:
@Luukdegram Thanks! I've updated the commit.